aws/logs_monitoring/parsing.py

```diff
@@ -266,6 +267,10 @@ def find_cloudwatch_source(log_group):
     if log_group.startswith("sns/"):
         return "sns"
 
+    # e.g. /aws/fsx/windows/xxx
+    if log_group.startswith("/aws/fsx/windows"):
```

Comment:
According to[ AWS official documentation](https://docs.aws.amazon.com/fsx/latest/WindowsGuide/file-access-auditing.html#faa-log-destinations):
```
The name of the CloudWatch Logs log group must begin with the /aws/fsx/ prefix. If you don't have an existing CloudWatch Logs log group when you create or update a file system on the console, Amazon FSx can create and use a default log stream in the CloudWatch Logs /aws/fsx/windows log group. 
```